### PR TITLE
Enhance deprecated argument documentation

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It improves documentation of deprecated arguments by adding a warning box

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     *Added limited documentation for renamed and removed variables*
     *Added warning message about deprecation*
 - Affected modules and functionalities:
     *Docs*
 - Key points relevant for the review:
     *Check the look and feel of docs?*
 - Validation and testing:
     *NA*
 - Documentation (including examples):
     *[Documentation updated]*

**JIRA TASK**: *[DALI-1894, DALI-1895]*

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

File: dali/python/nvidia/dali/ops.py [hunk 1]
```diff
@@ -97,17 +97,43 @@ def _get_kwargs(schema):
...
+        deprecation_warning = None
+        if schema.IsDeprecatedArg(arg):
+            meta = schema.DeprecatedArgMeta(arg)
+            msg = meta['msg']
+            assert msg is not None
+            deprecation_warning = ".. warning::\n\n    " + msg.replace("\n", "\n    ")
+            renamed_arg = meta['renamed_to']
+            # Renamed and removed arguments won't show full documentation (only warning box)
+            skip_full_doc = renamed_arg or meta['removed']
+            # Renamed aliases don't are not fully registered to the schema, that's why we query for the
```
Comment (banasraf) [inline dali/python/nvidia/dali/ops.py:113]:
```suggestion
            # Renamed aliases are not fully registered to the schema, that's why we query for the
```

**Review by banasraf — APPROVED** [review]
LGTM

File: dali/python/nvidia/dali/ops.py (hunk 1 above)
Comment (mzient) [reply dali/python/nvidia/dali/ops.py:113]: It's just a comment - we can skip CI if that's the going to be the only change in upcoming commit (just be sure not to `commit --amend`)

**Review by mzient — APPROVED** [review]
Looks OK. You can fix the comment, but that's not something I'd stop this from being merged.

**Pull request merged**